android/pytorch_android/src/androidTest/java/org/pytorch/PytorchTestBase.java

```diff
@@ -348,15 +348,32 @@ public void testChannelsLast3d() throws IOException {
   @Test
   public void testChannelsLastConv2d() throws IOException {
     long[] inputShape = new long[] {1, 3, 2, 2};
-    long[] dataNCHW = new long[] {1, 2, 3, 4, 11, 12, 13, 14, 101, 102, 103, 104};
+    long[] dataNCHW = new long[] {
+      111, 112,
```

Comment:
The change in values are to simplify debugging?

Reply:
Yes. I believe this makes the test more understandable, because you can actually **see** the convolution logic, which allowed me to infer what the 3d case would be. But I can ofc revert to simpler values if you want.